packages/cli/src/commands/source/push/add.spec.ts

```diff
@@ -0,0 +1,194 @@
+jest.mock('../../../lib/config/config');
+jest.mock('../../../hooks/analytics/analytics');
+jest.mock('../../../hooks/prerun/prerun');
+jest.mock('../../../lib/platform/authenticatedClient');
+jest.mock('@coveo/push-api-client');
+
+import {mocked} from 'ts-jest/utils';
+import {test} from '@oclif/test';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+import {DocumentBuilder, Source} from '@coveo/push-api-client';
+import {cwd} from 'process';
+const mockedClient = mocked(AuthenticatedClient);
+const mockedSource = mocked(Source);
+const mockedDocumentBuilder = mocked(DocumentBuilder);
+
+describe('source:push:add', () => {
+  const doMockAxiosError = (
+    status: number,
+    message: string,
+    errorCode: string
+  ) => ({
+    response: {
+      status,
+      data: {
+        errorCode,
+        message,
+      },
+    },
+  });
+
+  const doMockAxiosSuccess = (status: number, text: string) => ({
+    status,
+    statusText: text,
+  });
```

Comment:
Let's refactor that I think ;)

It's already in `delete.spec.ts`

Reply:
Indeed !